source/tests/endpoints/devices/test_devices.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import json
+
+from endpoints.devices.business_logic.Devices import BusinessLogicDevices
+from model.Database import Database
+from model.PluginManager import PluginManager
+from plugins.mock.light.Light import Light
+from plugins.mock.lock.Lock import Lock
+
+
+class TestDevices(unittest.TestCase):
+
+    def setUp(self):
+        self._database = Database()
+        self._plugin_manager = PluginManager()
+        self._logic = BusinessLogicDevices(self._database, self._plugin_manager)
+
+    def test_database_set_correctly(self):
+        self.assertEqual(self._database, self._logic._database)
+
+    def test_plugin_manager_set_correctly(self):
+        self.assertEqual(self._plugin_manager, self._logic._plugin_manager)
+
+    @unittest.skip("Currently not working because of singleton")
```

Comment:
Could you explain why this is not working?

If you add them to the database they should be in there isn't it.
Regardless if the database is singleton or not.